ow_ephemeris/scripts/sunlight_values_for_shaders.py

```diff
@@ -36,6 +36,7 @@ def computeIntensity(self):
 
     # Ignore missing transforms--there will probably be a few missing at startup
     try:
+      self.listener.waitForTransform(self.reference_body, self.sun, rospy.Time(0), rospy.Duration(20.0))
```

Comment:
Why did you have to add this line? Couldn't you just add a duration(20.0) param to the lookupTransform on next statement?